### ISSUE TITLE
Questions about the translations of BG 1 and 2, and also about the installation...

### ISSUE THREAD

**Issue by SilentMRG** [gemrb/gemrb#1950]
Hello everybody!

I have two localizations on my side for BG1 and 2 in Brazilian Portuguese. They were fan-made many years ago and work well in the vanilla versions, so I'd like to know, does the source port support translations for the aforementioned games?

And about the installation, please and if it's not inconvenient, I need a practical/direct guide on which files I should copy to the source port, so that the games work with the localizations (translations) mentioned above. I have the GOG classic versions, and unfortunately, they no longer work on Windows 7.

Thank you very much in advance!

**Comment by lynxlynxlynx** [comment]
Sure, translations work fine.
 I don't know what all those localizations entailed (eg. custom fonts, graphic updates?), but at minimum you need to copy over dialog.tlk and dialogf.tlk (if it exists). That will be the main thing, since that's where the text is stored.

**Comment by SilentMRG** [comment]
> Sure, translations work fine. I don't know what all those localizations entailed (eg. custom fonts, graphic updates?), but at minimum you need to copy over dialog.tlk and dialogf.tlk (if it exists). That will be the main thing, since that's where the text is stored.

Translations are just texts. They were released around 2005/2007 for vanilla versions and expansions. Yes, I have these files, but where should I put them? Thank you for your attention! =)

**Comment by lynxlynxlynx** [comment]
You just need to replace the ones shipped by gog.

**Comment by SilentMRG** [comment]
> You just need to replace the ones shipped by gog.

I think you didn't understand my question... I need to know where I should put the translation files as well as the game files in GemRB. This was my second question, as there is no direct explanation on the website. Anyway, I intend to build a portable version of BG 1 and 2 plus the translations on the source port. Is this possible or does GemRB only work with the retail/GOG version installed?

Cheers!

**Comment by lynxlynxlynx** [comment]
GemRB doesn't care where you put the game files, since it's configurable. It's the closest we have to the only obligatory config key, quite obviously mentioned here: https://gemrb.org/Install.html#configure-gemrb

I don't know what you mean by portable there though. If you want to play the game, the game files are needed.

**Comment by SilentMRG** [comment]
> I don't know what you mean by portable there though. If you want to play the game, the game files are needed.

In other words, for example: The source port of Diablo (DevilutionX) requires the ".MPQ" files inside the folder and together with "devilutionx.exe" so that it works portablely, that is, without the need for installation and creation of keys in the Windows registry, giving you the freedom to move the folder anywhere on the PC. Many source ports follow this rule and, sometimes, give the user a second option to move source port's binary (exe, for example) within the installed game to be able to play, and this is the easiest method, however, it is not portable.

To summarize even further...

I install the classic versions of GOG and move the "gemrb.exe" binary to the games' install folder along with the translations...

OR

I copy all the game files (the assets, music folder or files; texture folder or files, etc...) as well as the translations to the source port folder "gemrb-win32-v0.9.2"...

??

I hope I was clear because there is no other way to explain this, hehe.

**Comment by lynxlynxlynx** [comment]
Like I said, we don't care. You can have the game files wherever and GemRB files wherever (the latter even in several split locations).  

We don't use the windows registry and even if the official install process creates some keys, neither the original games nor us care — things can still be moved around. So I don't get it, even the originals are portable by this definition.

**Comment by SilentMRG** [comment]
Well then, I will install the games or extract them with innoextract and copy all the data that is inside the "app" folder to the folder where "gemrb.exe" is located, in this case, "gemrb-win32-v0.9.2 ". Honestly, I didn't know the vanilla version was portable, hehe.

Thank you very much for your attention and patience! =)
